### PR TITLE
feat: moved db from 2-multitenant to 5-appinfra

### PR DESCRIPTION
**Pull request by yliaog**

### CODE REVIEW

File: 5-appinfra/modules/env_baseline/db.tf [hunk 1]
```diff
@@ -17,58 +17,39 @@ module "cloudsql" {
...
+  ip_configuration = {
+    ipv4_enabled                  = false
+    psc_enabled                   = true
+    psc_allowed_consumer_projects = [var.cluster_project_id]
+  }
+
   additional_databases = [
     {
-      name      = "accounts-db"
+      name      = "${var.db_name}"
```
Comment (apeabody) [inline 5-appinfra/modules/env_baseline/db.tf:40]:
```suggestion
      name      = var.db_name
```

Comment (yliaog) [reply 5-appinfra/modules/env_baseline/db.tf]: done

File: 5-appinfra/modules/cicd-pipeline/locals.tf.sav [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Copyright 2024 Google LLC
```
Comment (gtsorbo) [inline 5-appinfra/modules/cicd-pipeline/locals.tf.sav:1]: @yliaog looks like there are duplicate files in the `cicd-pipeline/` folder - can you reconcile the `.sav` and `.bak` files?

**Review by gtsorbo — CHANGES_REQUESTED** [review]

File: 5-appinfra/modules/cicd-pipeline/locals.tf.sav (hunk 2 above)
Comment (yliaog) [reply 5-appinfra/modules/cicd-pipeline/locals.tf.sav]: sorry, added them by mistake, removed  now.

**Pull request merged**